src/main/resources/application.yml

```diff
@@ -111,7 +111,7 @@ interventions-ui:
       intervention-progress: "/probation-practitioner/referrals/{id}/progress"
       supplier-assessment: "/probation-practitioner/referrals/{id}/supplier-assessment"
       action-plan: "/probation-practitioner/referrals/{id}/action-plan"
-      session-feedback: "/probation-practitioner/action-plan/{id}/appointment/{sessionNumber}/post-session-feedback"
+      session-feedback: "/probation-practitioner/referrals/{id}/appointment/{sessionNumber}/post-session-feedback"
```

Comment:
Changed as requested to make url consistent with others. Is this url correct?

Reply:
sure - works for me. as long as the places where this is used have been updated to use the referral id instead of action plan id! 

has this route been added to the UI?